### PR TITLE
Added profile component

### PR DESCRIPTION
**Pull request by vanhornchadm**

### CODE REVIEW

File: client/src/containers/dashboard/dashboard.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export default class Dashboard extends React.Component {
           <SubmissionInfo />
         </div>
         <div className="group-info-grid">
-          <Profile />
+          <Profile teamName={"testTeam"} primaryContactName={"testName"} primaryContactEmail={"testEmail"} />
```
Comment (renodubois) [inline client/src/containers/dashboard/dashboard.js]: this looks good

File: client/src/containers/profile/profile.js [hunk 2]
```diff
@@ -4,8 +4,16 @@ export default class Profile extends React.Component {
   render() {
     return (
       <div>
-        <p>Replace me!</p>
+        <h1>{this.props.teamName} </h1>
+        <b>{this.props.primaryContactName} </b>
```
Comment (renodubois) [inline client/src/containers/profile/profile.js]: According to footnote on [this](https://www.w3schools.com/tags/tag_b.asp) page:
```
...the <b> tag should be used as a LAST resort when no other tag is more appropriate.
```
You can make some text bold by adding the css rule `font-weight: bold;`. Remember that adding style in React is a wee bit different, so check out the [this article](https://www.reactenlightenment.com/react-jsx/5.6.html) on how to do that. (note that this uses `var`, but you should be using `const` if you don't plan on changing that variable (likely the case here) or `let` if you want to change it)

That line should end up looking something like this:
```
<div style={/* insert style to make text bold here */}>{this.props.primaryContactName}</div>
```

File: client/src/containers/profile/profile.js [hunk 3]
```diff
@@ -4,8 +4,16 @@ export default class Profile extends React.Component {
...
+        <p><b>Email: </b>{this.props.primaryContactEmail} </p>
```
Comment (renodubois) [inline client/src/containers/profile/profile.js]: Refer to the above feedback on how to change this line as well.

File: client/src/containers/dashboard/dashboard.js (hunk 1 above)
Comment (renodubois) [inline client/src/containers/dashboard/dashboard.js]: To make this a little bit easier to transition to non test data, go ahead and create some variables for your test values. Something like:
```
const teamName = "testTeam"
```
Go ahead and do that for all 3 of the props you're pasting in. This way, we can easily transfer to using real data.

**Review by renodubois — CHANGES_REQUESTED** [review]
Here's my initial pass. Let me know if you have any questions, or if any of my comments don't make sense.

Also, before requesting the next review, go ahead and download [StandardJS](https://standardjs.com/index.html#install) and use the `standard` command to make sure all of your code matches the style guides set out there. Let me know if you have trouble installing that at all.

File: client/src/containers/profile/profile.js (hunk 2 above)
Comment (russleyshaw) [reply client/src/containers/profile/profile.js]: The irony is in that note, they use the `<b>` tag.
image

**Review by renodubois — APPROVED** [review]

**Pull request merged**